benchmark_runner/workloads/workloads_operations.py

```diff
@@ -85,7 +92,7 @@ def delete_all(self):
         This method delete all resources in namespace
```

Comment:
Might want to think about just deleting the namespace altogether, which will delete all the resources in it.

Reply:
This is what it did, delete all the resource in the namespace, but we must pass the resource name for it.

Reply:
No need.  Delete the entire namespace and all resources in that namespace go away.

Reply:
oc delete ns benchmark-runner